Generator_pgn.py

```diff
@@ -0,0 +1,82 @@
+import chess
+import torch.multiprocessing as mp
+import bz2
+
+class Generator_pgn():
+    def __init__(self,replay_queue: mp.Queue,pgn_path):
```

Comment:
format this file using black (https://github.com/psf/black) so that the styling is consistent. 

I recommend even adding a Bash script that automatically formats all Python code and running it before commit